src/communication/azext_communication/manual/_help.py

```diff
@@ -111,3 +111,58 @@
         text: |-
                az communication wait --name "MyCommunicationResource" --resource-group "MyResourceGroup" --deleted
 """
+
+helps['communication identity'] = """
+    type: group
+    short-summary: Manage identity with communication
+"""
+
+helps['communication identity issue-access-token'] = """
+    type: command
+    short-summary: "create user identity and accesstoken"
```

Comment:
Issues a new access token with the specified scopes for a given User Identity. If no User Identity is specified, creates a new User Identity as well.